services/cd-service/pkg/repository/repository.go

```diff
@@ -894,6 +899,25 @@ func (s *State) GetEnvironmentApplicationLocks(environment, application string)
 	}
 }
 
+func (s *State) GetDeploymentMetaData(environment, application string) (deployAuthor, deployTime string, err error) {
+	base := s.Filesystem.Join("environments", environment, "applications", application, "version", "environments", environment)
+	logs, err := readFile(s.Filesystem, s.Filesystem.Join(base, "deployment_metadata.yaml"))
+	if err != nil {
+		if os.IsNotExist(err) {
+			return "defaultUser", "", nil
```

Comment:
ahhh, I'm just refactoring the defaultUser.
for now, please use the constant in auth.go.
But depending on who merges first, we need to adapt this.